tests/core/shader.test.js

```diff
@@ -0,0 +1,261 @@
+/* global AFRAME */
+/* global assert, process, setup, suite, test, teardown */
+var entityFactory = require('../helpers').entityFactory;
+var componentName = 'shader';
```

Comment:
I know, but i did that as boilerplate so for the next component (since I did a series of them this way) I could clone the pattern and not have to keep changing "shader" with replace-all which is error-prone.